### ISSUE TITLE
Syntax error when requiring in the library

### ISSUE THREAD

**Issue by Jbarget** [prakhar1989/react-tags#64]
I'm getting the following error when adding the following line:

```js
const ReactTags = require('react-tag-input').WithContext;
```

```
SyntaxError: Unexpected token {
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:413:25)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/source/components/expedition/ExpeditionForm.tsx:5:19)
```

To give you a heads up our stack includes TypeScript. Any help would be great!

Thanks in advance

**Comment by sebinsua** [comment]
This is the same issue as: https://github.com/prakhar1989/react-tags/issues/63

**Comment by Jbarget** [comment]
@sebinsua thanks, are you suggesting reverting back to a version lower than 2.2.6 would fix it?

**Comment by sebinsua** [comment]
@Jbarget Reverting to `2.2.5` fixed the issue for my team.

**Comment by Jbarget** [comment]
@sebinsua thanks!

**Comment by prakhar1989** [comment]
Keeping this open so that I can fix the current release.

**Comment by prakhar1989** [comment]
Fixed in v2.2.8.

**Comment by Jbarget** [comment]
👍  nice
